### DISCUSSION TITLE
Circles Intro Sequence seems wrong with Screen Scaling

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#18810] [Q&A] [answered]
https://user-images.githubusercontent.com/64581009/175017763-4c5caec6-0ea2-4c91-b91c-22122187957b.mp4

Other intro sequences seems fine as it is.

**Comment by peppy** [comment, marked as answer]
Can you explain the perceived issue? It looks fine to me

**Reply by NotGumballer91** [reply]
Comparing the other intro sequences, Every other intro sequences is transparent and looks seamless:

https://user-images.githubusercontent.com/64581009/175019863-f2eb3eab-2b3a-4e93-9b82-b122e4606552.mp4


https://user-images.githubusercontent.com/64581009/175019887-9d52e4bb-15c9-4493-a852-7479fd34e1b7.mp4

While Circles intro suddenly draws huge black screen 

https://user-images.githubusercontent.com/64581009/175019926-8450fb3d-5fa1-42fd-abad-39f035b07280.mp4

**Reply by peppy** [reply]
If the three, the two you *didn't* report look more broken to me. "Welcome" is the most broken.

**Reply by NotGumballer91** [reply]
I don't know. "Welcome" looks fine as it is. "Triangles" have that clipping issue but not too bothered by it.
It's just that "Circles" has no transparency and showing a huge black screen instead of a being transparent is just unnatural to me.

**Reply by frenzibyte** [reply]
I would rather say the opposite, the intro shouldn't intersect with the background without a black layer behind it. Also with a transparent layer, the "Welcome" showing a black logo instead of transparent looks quite off to me.

**Reply by peppy** [reply]
That black screen is your game screen.



> The "Welcome" showing a black logo looks quite off to me.

Precisely.

The fix would be adding black backgrounds to the other intros and calling it a day.

**Reply by NotGumballer91** [reply]
> The fix would be adding black backgrounds to the other intros and calling it a day

Oh that's unfortunate. I personally liked the transparency with the bg behind it but oh well. If you're going by that, make sure to add black bg to the "This is osu!" screen as well. For consistency.
